### DISCUSSION TITLE
【读书会】富爸爸穷爸爸 - 第1期 | ApacheCN

### DISCUSSION THREAD

**Discussion by yanqiang921106** [apachecn/home#230] [Ideas]
# <center>【读书会】富爸爸穷爸爸 - 第1期 | ApacheCN<center>

* 【推荐】读书会: 《富爸爸穷爸爸》罗伯特·清崎
* 【推荐】留言地址: https://github.com/apachecn/home/issues/189

> 参与

* QQ群【临时群】: 1062944139
* 负责人紫色木槿: 1076003661 (QQ)

> 时间

**每两周三晚8点，月初会发布本月阅读书籍2-4本，顺序阅读。

* 时间周期: 2021-05-20 ～ 2021-06-02 （共 14 天，对自己严格的建议控制在7天内）
* 语音频率: 每两周三晚9点语音一次，周末休息跳过
* **群友发言：**5~10人，5分钟/人，计划时长0.5-1小时	
* **[@紫色木槿](https://github.com/yanqiang921106)** 会进行语音: 沟通进度和督促学习

> 内容

1. 分享读书心得
2. 学到的知识
3. 可以应用到实际工作生活中的
4. 自己的应用计划			

每一本书，专门一个issues，供大家留言分享		

> 活动目的

1. 培养自己对理财的正确认识
2. 选出2个人作为大肥羊学校的学员，作为其他活动的代理班长		

---

* 总结：一人可以走的很快，但是一群人却可以走的很远

**Comment by jiangzhonglian** [comment]
教东西一定要说或讲吗?
一定要:
醒醒，这是重点，做笔记?

**Comment by jiangzhonglian** [comment]
很多人说：“我对钱没兴趣”，可他们却每天都工作8小时。

**Comment by jiangzhonglian** [comment]
你们俩是最先请求我教你们赚钱的人，我有150多个雇员，但没有一个人问过我这个问题。他们只是要求工作和报酬，我从来没有教过他们关于金钱的知识。他们把一生中最好的年华用来挣钱，却不明白到底是为了什么而工作。

**Comment by jiangzhonglian** [comment]
工作只是试图用暂时的办法来解决长期的问题。

**Comment by jiangzhonglian** [comment]
造成贫困和财务问题的主要原因是恐惧和无知，而不是经济环境、政府或者富人。

**Comment by jiangzhonglian** [comment]
资产是能把钱放进我口袋里的东西。
负债是把钱从我的口袋里取走的东西。

**Comment by jiangzhonglian** [comment]
中产阶级如何摆脱财务问题？

把钱投在那些能带来收入的资产上。

**Comment by jiangzhonglian** [comment]
作为一个自已有房子的雇员，你努力工作的结果如下:

1. 你为公司工作。雇员的工作只会使雇主成股东更加富有而不能使他们自己富有。你的努力和成功将便雇主更加成功并得以提早退休。
2. 你为政府工作。政府在你还未看到自己的工资时就已拿走了一部分，你努力工作，其结果是使政府的税收增加。实际上大多数人每年从1月到5月都是在为政府白白工作。
3. 你为银行工作。缴了税后，你的最大支出应该是抵押贷款和信用卡账单了。


*你要学会怎样才能让你的努力更直接的为你和你的家人带来更多好处呢？*

**Comment by jiangzhonglian** [comment]
存在财务问题的人经常耗费一生为别人工作。

**Comment by jiangzhonglian** [comment]
真正的资产，分为以下几类:

- 不需我到场就可以正常运作的业务。我拥有它们。但由别人经营和管理。如果我必须在那儿工作，那它就不是我的事业而是我的职业了。
- 股票
- 债券
- 能够产生收入的房地产
- 票据(借据)
- 版税。 如音乐、手稿、专利
- 其他任何有价值、 可产生收入或有增值潜力并且有很好销路的东西

**Comment by jiangzhonglian** [comment]
关注自己的事业，意思是: 建立自己牢固的资产。
一旦1美元投入到资产项，那么它就成了你的雇员。
你可以让它一天24小时不间断的工作，努力为你服务。

**Comment by tenghf** [comment]
现在普遍是你应该先努力学习，再找到一份好的稳定的工作，为了钱而工作，而没有好好的利用钱来为你工作
应该培养财商，提高资产复利投资能力

**Comment by yanqiang921106** [comment]
开创事业所必备的最重要的3种管理技能是：
1．现金流管理。
2．人事管理。
3．个人时间管理

**Comment by yanqiang921106** [comment]
但掌握财务知识的人有时候还
是不能积累丰厚的资产项，其主要原因有5个：
1．恐惧心理。
2．愤世嫉俗。
3．懒惰。
4．不良习惯。
5．自负

**Comment by jiangzhonglian** [comment]
我常常吃惊为什么有些人才华过人却只能挣到很低的工资。
我也曾遇见聪明且受过高等教育的人每年收入不到2万美金。

**Comment by supermanwasd** [comment]
1.讲了很多方法论，写成功日记增加信心，找到自己擅长的点来赚钱，基金是怎么回事。
2.不要为失去的东西而忧伤，而要对拥有它的时光心存感激。

**Comment by jiangzhonglian** [comment]
1
